README.md

```diff
@@ -2,59 +2,127 @@
+1.  Enable the [Genomics, Compute Engine, Cloud Storage, and Dataflow APIs](https://console.cloud.google.com/flows/enableapi?apiid=genomics,storage_component,storage_api,compute_component,dataflow)
+1.  Open the [billing](https://console.cloud.google.com/project/_/settings) page
+    for the project you have selected or created, and click _Enable billing_.
+1.  Create a new BigQuery dataset by visiting the
+    [BigQuery web UI](https://bigquery.cloud.google.com/), clicking on the
+    down arrow icon next to your project name in the navigation, and clicking on
+    _Create new dataset_.
+
+## Loading VCF files to BigQuery
+
+### Using docker
+
+The easiest way to run the VCF to BigQuery pipeline is to use the
+[docker](https://www.docker.com/) image and run it with the
+[Google Genomics Pipelines API](https://cloud-dot-devsite.googleplex.com/genomics/pipelines)
+as it has the binaries and all dependencies pre-installed.
+
+First, set up the pipeline configurations shown below and save it as
+`vcf_to_bigquery.yaml`. The parameters that you need to replace are:
+
+* `my-project`: This is your project name that contains the BigQuery dataset.
+* `gs://my-bucket/vcffiles/*.vcf`: A location in Google Cloud Storage where the
+  VCF file are stored. You may specify a single file or provide a pattern to
+  load multiple files at once. Please refer to the
+  [Variant Merging](docs/variant_merging.md) documentation if you want
+  to merge samples across files. The pipeline supports gzip, bzip, and
+  uncompressed VCF formats. However, it runs slower for compressed files as they
+  cannot be sharded.
+* `my-bigquery-dataset`: Your BigQuery dataset to store the output.
+* `my-bigquery-table`: This can be any ID you like (e.g. vcf_test).
+* `gs://my-bucket/staging` and `gs://my-bucket/temp`: These can be any folder in
+  Google Cloud Storage that your project has write access to. These are used to
+  store temporary files needed for running the pipeline.
+
+```yaml
+name: vcf-to-bigquery-pipeline
+docker:
+  imageName: gcr.io/gcp-variant-transforms/gcp-variant-transforms
+  cmd: |
+    ./opt/gcp_variant_transforms/bin/vcf_to_bq \
+      --project my-project \
+      --input_pattern gs://my-bucket/vcffiles/*.vcf \
+      --output_table my-project:my-bigquery-dataset.my-bigquery-table \
+      --staging_location gs://my-bucket/staging \
+      --temp_location gs://my-bucket/temp \
+      --job_name vcf-to-bigquery \
+      --runner DataflowRunner
+```
+
+Next, run the following command to launch the pipeline. Replace `my-project`
+with your project name, `gs://my-bucket/temp/runner_logs` with a Cloud Storage
+folder to store the logs from the pipeline.
+
+```bash
+gcloud alpha genomics pipelines run \
+    --project my-project \
+    --pipeline-file vcf_to_bigquery.yaml \
+    --logging gs://my-bucket/temp/runner_logs \
+    --zones us-west1-b \
+    --service-account-scopes https://www.googleapis.com/auth/bigquery
```

Comment:
Placing "<>" around parameters that need to be replaced can be helpful to users.

Reply:
See above.